jdat_session/MosvizJwebbinarDemoRequirements.txt

```diff
@@ -0,0 +1,3 @@
+decorator==4.4.2
+astropy==4.2.1
+jdaviz==1.1
```

Comment:
```suggestion
decorator>=4.4.2
astropy>=4.2.1
jdaviz>=1.1
```
I suggest schanging this to "lower pinnings", because I suspect most users are not going to be wanting to have a stand-alone webbinar environment.,

Reply:
Hmm interesting; I imagined we would be asking users to start in a fresh environment for reproducibility purposes. If users are bringing in their own environments, couldn't that be a massive headache if things break?

Reply:
We will not support their own environments for the webinar. They will all work on the AWS platform.

Reply:
Actually, if that's the case, should we be testing our notebooks against a specific environment @camipacifici?